### PR TITLE
ui2 management and setup screens

### PR DESCRIPTION
**Pull request by alexmatson-da**

### CODE REVIEW

File: ui2/public/favicon.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <style>
+    path {
+      stroke: #3A465F;
+    }
+    @media (prefers-color-scheme: dark) {
+      path {
+        stroke: #4BE725;
+      }
+    }
+</style>
```
Comment (alexmatson-da) [inline ui2/public/favicon.svg:11]: (This changes the tab favicon color depending on the system color scheme!)

Comment (alexgraham-da) [reply ui2/public/favicon.svg:11]: nice!

File: ui2/src/components/InputDialog/InputDialog.tsx [hunk 2]
```diff
@@ -87,6 +88,7 @@ export function InputDialog<T extends { [key: string]: any }>(props: InputDialog
         <Button className="ghost" onClick={() => props.onClose(null)}>
           Cancel
         </Button>
+        {props.error && <p className="error">{props.error}</p>}
```
Comment (Sarahb-da) [inline ui2/src/components/InputDialog/InputDialog.tsx:91]: we could add in `AppError` to this code at some point as well

**Review by Sarahb-da — APPROVED** [review]
Looks good to me. Great job, Alex

**Review by brian-weir — APPROVED** [review]

**Comment by brian-weir** [comment]
lgtm - great work @alexmatson-da !!

**Review by alexgraham-da — APPROVED** [review]
very nice, looks good to me!

File: ui2/src/components/InputDialog/InputDialog.tsx (hunk 2 above)
Comment (alexmatson-da) [reply ui2/src/components/InputDialog/InputDialog.tsx:91]: agreed, reintroducing the standard error handling we had before is going to be a future pr for sure :)

**Pull request merged**